app/controllers/miq_ae_class_controller.rb

```diff
@@ -188,7 +189,9 @@ def get_node_info(node)
         set_right_cell_text(x_node, @record)
       end
     else
-      @grid_data = User.current_tenant.visible_domains
+      visible_domains = User.current_tenant.visible_domains
+      @grid_data = visible_domains
+      add_all_domains_version_message(visible_domains)
```

Comment:
The `visible_domains` variable does not server any purpose here, just pass `@grid_data` into the `add_all_domains_version_message` method.